DDECal/DDECal.cc

```diff
@@ -306,10 +306,11 @@ void DDECal::initializeConstraints(const ParameterSet& parset, const string& pre
 
 void DDECal::initializeIDG(const ParameterSet& parset, const string& prefix)
 {
-  std::string
-    imageFilename = parset.getString(prefix + "idg.image"),
-    regionFilename = parset.getString(prefix + "idg.regions");
-  itsFacetPredictor.reset(new FacetPredict(imageFilename, regionFilename));
+  std::string regionFilename =
+    parset.getString(prefix + "idg.regions");
+  std::vector<std::string> imageFilenames =
+    parset.getStringVector(prefix + "idg.images");
```

Comment:
New usage, not backward compatible (but that's probably no problem for the few people that now use IDG in DPPP). Please document at https://www.astron.nl/lofarwiki/doku.php?id=public:user_software:documentation:ndppp

Reply:
Yes that was still missing, thanks, now added to the docs.